### DISCUSSION TITLE
captionLayout that includes dropdowns and arrow buttons

### DISCUSSION THREAD

**Discussion by iantbarker** [gpbl/react-day-picker#1526] [Ideas]
Hello,

When I added `captionLayout="dropdown"`, I got month and year dropdowns, but I lost the next and previous arrows. Is there a way to have both? 

Ian

**Comment by luizfm** [comment]
Up! Is there any motive for that? I like the general idea of having both options. 

Users do like to have the possibility of selecting through the months / years, but also quick options to navigate between them using arrows

**Reply by iantbarker** [reply]
I think so, yes. I feel like a lot of date pickers have both options available and would be nice to have.

**Comment by gpbl** [comment]
Working on it! https://github.com/gpbl/react-day-picker/issues/1663
